base/java-tools/src/com/netscape/cmstools/system/SecurityDomainCLI.java

```diff
@@ -41,6 +41,7 @@ public SecurityDomainCLI(MainCLI mainCLI) {
         this.mainCLI = mainCLI;
 
         addModule(new SecurityDomainShowCLI(this));
```

Comment:
The `pki securitydomain-show` displays a mixture of the domain, subsystems, and hosts information. It's not consistent with the rest of the CLIs. Normally the CLIs follows this pattern: `pki <object>-<CRUD operations>`. The new `pki securitydomain-host-*` CLIs are more consistent in that regard because they are specifically provided to manage the security domain hosts. Eventually we might want to deprecate/drop `pki securitydomain-show` in the future.